### DISCUSSION TITLE
Is there anyway to change toolbar section of the DateRangePicker?

### DISCUSSION THREAD

**Discussion by gaurav-wps** [rsuite/rsuite#2163] [Q&A]
I have this business need where I have to add custom input functions in the place of toolbar(bottom) area of the DateRangePicker along with it, I also have to changed the text and layout of "OK" button. 

P.s. I am new to rsuite. Quick answer will be appreciated.

**Comment by SevenOutman** [comment]
> I have this business need where I have to add custom input functions in the place of toolbar(bottom) area of the DateRangePicker along with it

Hi @gaurav-wps, if what you need is what we call "shortcuts", you can achieve it with the `ranges` prop. See related docs here https://rsuitejs.com/components/date-range-picker/#custom-shortcut-options

> I also have to changed the text and layout of "OK" button.

The OK button is not customizable at this moment, but I think it's a nice feature to implement.

**Comment by kaaviya-min** [comment]
@SevenOutman It's been 3 years. Is there any better way to this? Like add a footer?
